third-party/realsense-file/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8.9)
```

Comment:
So you allow max future cmake version of 3.20.5?
Just want to make sure I understand the syntax.

Reply:
It means it's been tested and will work with anything up to 3.20.5, even if earlier versions are deprecated

Reply:
(while still keeping a minimum of 2.8.9)

Reply:
Got it 👍